product_variant_weight/README.rst

```diff
@@ -0,0 +1,52 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Product Variant Weight
+======================
+
+This module allows to have product weight at product level instead of
+product template
```

Comment:
period (`.`) at the end. Include a usage section about the fact that writing these fields on the template, they propagate to the variants.